components/edit-collective/EditHostSettings.js

```diff
@@ -38,158 +41,79 @@ const PlanPrice = styled.p`
-                <ul>
-                  <li>Up to 10 hosted collectives</li>
-                  <li>Unlimited added funds</li>
-                  <li>Manual bank transfers</li>
-                  <li>Host dashboard</li>
-                </ul>
-              </PlanFeatures>
-              <PlanPrice>$50 / month</PlanPrice>
-
-              {/* Staging */}
-              <Button href={`/opencollective/contribute/medium-host-plan-14969/checkout?redirect=${redirectUrl}`}>
-                Upgrade
-              </Button>
-            </Plan>
-
-            <Plan>
-              <PlanName>Large Host Plan</PlanName>
-              <PlanFeatures>
-                <ul>
-                  <li>Up to 25 hosted collectives</li>
-                  <li>Unlimited added funds</li>
-                  <li>Manual bank transfers</li>
-                  <li>Host dashboard</li>
-                </ul>
-              </PlanFeatures>
-              <PlanPrice>$100 / month</PlanPrice>
-
-              {/* Staging */}
-              <Button href={`/opencollective/contribute/large-host-plan-14967/checkout?redirect=${redirectUrl}`}>
-                Upgrade
-              </Button>
-            </Plan>
-
-            <Plan>
-              <PlanName>Network Host Plan</PlanName>
-              <PlanFeatures>
-                <ul>
-                  <li>Unlimited hosted collectives</li>
-                  <li>Unlimited added funds</li>
-                  <li>Manual bank transfers</li>
-                  <li>Host dashboard</li>
-                </ul>
-              </PlanFeatures>
-              <PlanPrice>$100+ / month</PlanPrice>
-              <Button disabled>Contact Us</Button>
-            </Plan>
-          </tr>
-        </Table>
-
-        <h3>
-          <FormattedMessage id="collective.hostSettings.currentPlan.title" defaultMessage="Current Plan Usage" />
-        </h3>
-
-        <ul>
-          <li>
-            <strong>Current Plan</strong>: <Capitalized>{collective.plan.name}</Capitalized>
-          </li>
-          <li>
-            <strong>Collective Limit</strong>: {collective.plan.hostedCollectives} of{' '}
-            {collective.plan.hostedCollectivesLimit}
-          </li>
-          <li>
-            <strong>Added Funds Limit</strong>:&nbsp;
-            {collective.plan.addedFundsLimit && (
-              <span>
-                ${collective.plan.addedFunds / 100} of ${collective.plan.addedFundsLimit / 100}
-              </span>
-            )}
-            {!collective.plan.addedFundsLimit && (
-              <FormattedMessage id="collective.hostSettings.addedFunds.unlimited" defaultMessage="Unlimited" />
-            )}
-          </li>
-          <li>
-            <strong>Host Dashboard</strong>: {collective.plan.hostDashboard ? 'Yes' : 'No'}
-          </li>
-        </ul>
-      </div>
-    );
-  }
-}
+const EditHostSettings = props => {
+  const { collective } = props;
+  const { data: opencollective } = useQuery(getCollectiveQuery, {
```

Comment:
Can you use a custom query instead of the `getCollectiveQuery` ?

Reply:
And make sure we query `longDescription` on Tier.

Reply:
Sure, I'll write a custom query in this file.